mapiproxy/servers/default/emsmdb/oxcfxics.c

```diff
@@ -790,15 +791,17 @@ static void oxcfxics_table_set_cn_restriction(struct emsmdbp_context *emsmdbp_ct
 static bool oxcfxics_push_messageChange(struct emsmdbp_context *emsmdbp_ctx, struct emsmdbp_object_synccontext *synccontext, const char *owner, struct oxcfxics_sync_data *sync_data, struct emsmdbp_object *folder_object)
 {
 	TALLOC_CTX			*mem_ctx, *msg_ctx;
-	bool				folder_is_mapistore, end_of_table, changed_prop_index = false;
+	bool				folder_is_mapistore, changed_prop_index = false;
+	bool				end_of_table = false;
 	struct emsmdbp_object		*table_object, *message_object;
 	uint32_t			i;
-	static enum MAPITAGS		mid_property = PidTagMid;
 	enum MAPISTATUS			*retvals, *header_retvals, retval_msg_class = MAPI_E_NOT_FOUND;
+	const enum MAPITAGS		table_prop_tags[] = {PidTagMid, PidTagChangeNumber};
 	void				**data_pointers, **header_data_pointers;
 	struct FILETIME			*lm_time;
 	NTTIME				nt_time;
 	uint32_t			unix_time, contextID, message_class_prop_index;
+	const uint32_t			TABLE_PROPS_COUNT = sizeof(table_prop_tags) / sizeof(enum MAPITAGS);
```

Comment:
I agree that this should be downcase (I know there are functions with variables like this, upcase, but I never liked it).